### PR TITLE
Ability to run plugin tests with ci runner script

### PR DESCRIPTION
**Pull request by basiszwo**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://run-plugin-tests-on-ci.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://run-plugin-tests-on-ci.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: lib/tasks/cucumber.rake [hunk 1]
```diff
@@ -72,6 +72,18 @@ unless ARGV.any? { |a| a =~ /\Agems/ } # Don't load anything when running the ge
             features = args[:features].split(/\s+/)
           else
             features = get_plugin_features
+
+            if name == :plugins
+              # in case we want to run cucumber plugins are there are none
```
Comment (machisuji) [inline lib/tasks/cucumber.rake]: >  in case we want to run cucumber plugins ~~are~~ **and** there are none

typo

File: script/ci_runner.sh [hunk 2]
```diff
@@ -53,8 +53,7 @@ run() {
   eval $3;
 }
 
-if [ $2 != '' ] && [ $3 != '' ]
-then
+if [ $2 != '' ] && [ $3 != '' ]; then
```
Comment (machisuji) [inline script/ci_runner.sh]: @basiszwo If this triggers warning\* maybe a better way to check those vars are not empty would be the following one:

``` bash
if [ -n "$2" ] && [ -n "$3" ]; then
```

\* `script/ci_runner.sh: line 56: [: !=: unary operator expected`

`-n` would be such a unary operator I guess.

Comment (basiszwo) [inline script/ci_runner.sh]: :+1:

**Pull request merged**